kratos/geometries/geometry.h

```diff
@@ -1122,41 +1122,102 @@ class Geometry : public PointerVector<TPointType>
         return false;
     }
 
+    ///@}
+    ///@name Boundaries
+    ///@{
+
+    /**
+     * @brief This method gives you all boundaries entities of this geometry.
+     * @details This method will gives you all the boundaries entities
+     * @return GeometriesArrayType containes this geometry boundaries entities.
+     * @see GeneratePoints()
+     * @see GenerateEdges()
+     * @see GenerateFaces()
+     */
+    virtual GeometriesArrayType GenerateBoundariesEntities( void )
+    {
+        const SizeType dimension = this->LocalSpaceDimension();
+        if (dimension == 3) {
+            return this->GenerateFaces();
+        } else if (dimension == 2) {
+            return this->GenerateEdges();
+        } else if (dimension == 1) {
+            return this->GeneratePoints();
+        }
+
+        return GeometriesArrayType();
```

Comment:
I would make the last else if -> else

so that we can avoid this last line

Reply:
Okidoki